src/lucky/tags/specialty_tags.cr

```diff
@@ -74,4 +75,14 @@ module Lucky::SpecialtyTags
     how_many.times { raw("&nbsp;") }
     view
   end
+
+  private def build_content_attr_value(options)
```

Comment:
Could you name this something like `build_responsive_meta_tag_content_attr`. So that it is clear it is for the responsive_meta_tag? We may need something like this in the future for other methods, but until then I think being specific will add clarity

Reply:
Would `build_viewport_properties` be ok? If so, than I think it's ready.